migrations/20231110114933-fix-payment-processor-fee-cover.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`
+     UPDATE "Transactions"
+     SET "deletedAt" = NOW()
```

Comment:
More of a personal preference: I like to mark migrated transactions somehow; I think it helps to investigate later when we stumble on the data and wonder why it's in a certain state.

```suggestion
     SET "deletedAt" = NOW(), "data" = jsonb_set(COALESCE("data", '{}'), '{deletedFromMigration20231110114933}', 'true')
```